app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelMain.tsx

```diff
@@ -103,6 +132,8 @@ const ConnectionPanelMain = ({
 					actionSubject: 'tab'
 				}
 			);
+		} else {
+			setShowSharePageContainer(false);
```

Comment:
ive not done a lot of react, so unsure, but do we need this else? looks like if defaults to false anyway

Reply:
it would be required since clicking other 'tab' in the UI, this SharePageContainer would still persist. But we want it to appear and disappear the same way as the SetUpGuide tab